apps/OboeTester/app/src/main/java/com/mobileer/oboetester/TestAudioActivity.java

```diff
@@ -663,8 +663,8 @@ public void startAudio() throws IOException {
         Log.i(TAG, "startAudio() called =========================");
         int result = startNative();
         if (result != 0) {
-            showErrorToast("Start failed with " + result);
-            throw new IOException("startNative returned " + result);
+            showErrorToast("Start failed with " + StreamConfiguration.convertErrorToText(result));
```

Comment:
It would be handy to also include the error code because some people are used to the numbers and some QA test procedures refer to the number. Eg:

    " Start failed with -869, ErrorClosed"